src/picongpu/submit/hypnos-hzdr/PBS_Tutorial.rst

```diff
@@ -0,0 +1,45 @@
+PBS examples
+============
+
+Job Submission
+""""""""""""""
```

Comment:
QUESTION:
In #2005 you removed double quotes and replaced them by single quotes.
Why do you use double quotes here?